src/CompositionRoot.ts

```diff
@@ -21,7 +21,7 @@ export function getCompositionRoot(instance: Instance) {
         }),
         metadata: getExecute({
             listAll: new GetAllMetadataUseCase(metadataRepository),
-            list: new GetMetadataUseCase(metadataRepository),
+            list: new GetMetadataWithDependenciesUseCase(metadataRepository),
```

Comment:
Consistently rename the usecase caller

```suggestion
            getDependencies: new GetMetadataWithDependenciesUseCase(metadataRepository),
```